src/main/webapp/WEB-INF/jsp/editCalendarDefinition.jsp

```diff
@@ -74,7 +74,7 @@
     </div>
 
     <div class="row" role="main">
-        <portlet:actionURL var="postUrl"><portlet:param name="action" value="editCalendarDefinition"/></portlet:actionURL>
+        <portlet:actionURL var="postUrl" escapeXml="false"><portlet:param name="action" value="editCalendarDefinition"/></portlet:actionURL>
```

Comment:
Is there any risk of HTML injection disabling the escaping?

Reply:
:+1: Needed to differentiate between characters used (i.e. &) and those that are already escaped in URL

Reply:
The value is rendered only in the edit URL forms. The value placed in the JSP via a form:input tag. This can be escaped with `htmlEscape="true"` to `form:input`. Because the raw value is needed to correctly set the URLs (so it seems), this un-escaped text is needed. It is also limited in that only the author editing the value would be exposed.